### PR TITLE
Fixes #31 and adds spaces in README.md

### LINKED ISSUE #31: Add Code of Conduct badge
Add code of conduct badge, a la https://github.com/tessel/project/pull/190

### PR DESCRIPTION
**Pull request by brihijoshi**
Added Code of Conduct, along with removing spaces to produce proper Markdown titles. Also, removed the # for * in bullet points.

### CODE REVIEW

**Comment by Frijol** [comment]
Fixes https://github.com/tessel/rfid-pn532/issues/31

File: README.md [hunk 1]
```diff
@@ -29,43 +31,43 @@ rfid.on('ready', function (version) {
 });
 ```
 
-###Methods
+### Methods
 
-&#x20;<a href="#api-rfid-setPollPeriod-pollPeriod-callback-err" name="api-rfid-setPollPeriod-pollPeriod-callback-err">#</a> rfid<b>.setPollPeriod</b>( pollPeriod, callback(err) )  
- Set the time in milliseconds between each check for an RFID device. The poll period is 500ms by default.
+* &#x20;<a href="#api-rfid-setPollPeriod-pollPeriod-callback-err" name="api-rfid-setPollPeriod-pollPeriod-callback-err"></a> rfid<b>.setPollPeriod</b>( pollPeriod, callback(err) )
```
Comment (Frijol) [inline README.md]: These are actually done in this way intentionally– it lets us link to specific methods in the API e.g. https://github.com/tessel/rfid-pn532#api-rfid-on-read-callback-data

**Comment by brihijoshi** [comment]
Sincere Apologies. Changes to be done back in the remaining repositories ASAP

File: README.md (hunk 1 above)
Comment (brihijoshi) [inline README.md]: I get the usage of the #s now. Fixed in all the repos.

**Pull request merged**